gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -1,17 +1,18 @@
-# FIXME:
-#   1. Edit foreman-proxy requirement and specify minimum version
-#   2. Delete these lines
-#
+# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
 # template: smart_proxy_plugin
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
 %global gem_name <%= spec.name %>
 %global plugin_name <%= spec.name.sub(/\Asmart_proxy_/, '') %>
 
-%global foreman_proxy_min_version FIXME
-%global foreman_proxy_dir %{_datarootdir}/foreman-proxy
+%global foreman_proxy_min_version 1.24
```

Comment:
Is this chosen because that will change from non-SCL to SCL or as a sort of safe default that we should update with every release?

Reply:
Yessir.